plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile

```diff
@@ -445,31 +445,30 @@ p=. Orthogonal states
 
 h3(#sclang_shallow_history_states). Shallow history states
 
-A *shallow history state* is a pseudo state that is placed inside the region of a composite state. It is used to remember the last active state inside a composite state. This makes it possible to jump back to the remembered state instead of starting at the inner initial state again.
+A *shallow history state* is a pseudo state that is placed inside the region of a composite.
+
+
 
 A history state saves the current state exactly when its containing region is left. A shallow history state saves the state of the current level only. If you need to save the status of multiple nested composite states, use a "deep history state":#sclang_deep_history_states.
 
 ==<div class="example">==
 
-###. FIXME: Re-create this example with nice names and even nicer drawings!
-
 The following example, showing the answering of a questionaire, explains this:
 
-!images/docu_shallowHistory01.jpg(Shallow history [1])!
+!images/docu_shallow_history_init.png(Shallow history [1])!
+!images/docu_shallow_history_nextQuestion1.png(Shallow history [2])!
+!images/docu_shallow_history_nextQuestion2.png(Shallow history [3])!
```

Comment:
Filenames are case-sensitive! Please always use lower-case, including the filename extension!

Reply:
I also optimized the image file sizes.

Reply:
The images need visual improvents in order to look nice:
* The *pause* transition arrow should end in the middle of the **init** state, not at its low-left corner.
* The **q** states should be of the same size.
* The **q** states should be properly aligned, e.g., vertically.
* The **init** state should have the same size as the **q** states.